components/reports/precipitation/ReportPrecipChart.vue

```diff
@@ -168,8 +194,37 @@ export default {
 					},
 				],
 				hovermode: 'x unified',
+				annotations: [],
+				legend: {
+					x: 1.03
+				},
+				margin: {
+					b: 120
+				},
+				height: 500,
 			}
 
+			let footer_y = -0.4
+			if (window.innerWidth < 1250) {
+				layout['xaxis'] = {
+					tickangle: 45,
+				}
+				layout['margin']['b'] = 160
+				footer_y = -0.65
+			}
+
+			layout.annotations.push({
+				x: 0.5,
+				y: footer_y,
+				xref: 'paper',
+				yref: 'paper',
+				showarrow: false,
+				text: 'The boxplot represents the interquartile range (IQR) of historical means for the season, from 1950-2009.<br /> \
+					The shaded gray region shows the extent of common variation for the historical period.<br /> \
+					Credit: Scenarios Network for Alaska + Arctic Planning, University of Alaska Fairbanks.<br /> \
+					Visit northernclimatereports.org for more information.',
```

Comment:
I think for now, drop the Credit and site link from the table.  Credit might need to be something different (USGS, CASC, something) and I think it's OK to omit the "link back" -- encouraging attribution can be a little tricky, and we may want to add something to the tool which has some suggestions.  There are standard citation formats for web sites, figures, etc, that people _should_ be using, too.

Reply:
Chart footer credits & URL have been removed in 9d178c97.